contrib/win32/win32compat/pwd.c

```diff
@@ -262,6 +262,50 @@ w32_getpwuid(uid_t uid)
 	return ret;
 }
 
+PSID
+getusid(void)
+{
+	HANDLE token = 0;
+	DWORD info_len = 0, length = 0;
+	TOKEN_USER* info = NULL;
+	PSID user_sid = NULL;
+
+	errno = 0;
+	if (OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &token) == FALSE) {
+		debug2("OpenProcessToken() failed. Error code is : %d.\n", GetLastError());
```

Comment:
Remove new line "\n" for each of the trace messages. debug* will take care of it automatically